3rdPartyCypress/cypress/integration/worldServiceCoUkRedirect.js

```diff
@@ -0,0 +1,36 @@
+import services from '../../../src/app/lib/config/services';
+
+Object.keys(services).forEach(service => {
+  const notWSServices = ['news', 'cymrufyw', 'naidheachdan', 'default']; // Not WS
+
+  if (notWSServices.includes(service)) {
+    return;
+  }
+  describe('WS Redirects', () => {
+    it(`should redirect *bbc.com/${service}`, () => {
+      const urlsTotest = [
+        `https://www.bbc.co.uk/${service}`,
+        `https://www.bbc.co.uk/${service}/articles/a0000000000o`,
+        `https://www.bbc.co.uk/${service}/articles/a0000000000o.amp`,
+      ];
+
+      urlsTotest.forEach(urlToTest => {
+        const slashLoc = urlToTest.indexOf('/', 8);
+        cy.request({
+          url: urlToTest,
+          followRedirect: false,
+        }).then(resp => {
+          expect(resp.status).to.eq(301);
+          // expect first slice to equal https://www.bbc.com/
+          expect(resp.redirectedToUrl.substring(0, 20)).to.eq(
+            'https://www.bbc.com/',
+          );
+          // expect second slice to equal whatever came in
+          expect(resp.redirectedToUrl.substring(20)).to.eq(
```

Comment:
I'd suggest using `startsWith(...)` and `endsWith` over `substring(...)`.

Reply:
@mykeels feel free to make the change, I don't feel it really matters in an e2e test

Reply:
I just re-read the test, and `substring(...)` makes for a better assertion in this case, as you can read the assertion result/error and know immediately what two strings weren't equal.